Documentation/v3/rkt_guide.md

```diff
@@ -0,0 +1,79 @@
+
+# Running etcd under rkt
+
+The following guide will show you how to run etcd under rkt using the [static bootstrap process](clustering.md#static).
+
+## Running a standalone node
```

Comment:
I was confused about this. can you say single node or something like that? or change the description to say "running a single node, which is useful for X or Y"